net/ftpproxy/+POST_DEINSTALL

```diff
@@ -0,0 +1,28 @@
+DIFF='--- filter.inc.ftpproxy 2016-09-21 16:38:53.947075272 +0200
```

Comment:
For now the plugin is marked private, it'll build but not show up in the plugins list. Since 16.7.5 builds are currently running we will have time to fix this until 16.7.6 :)